src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -751,7 +738,7 @@ SubCmdProgram::execute(const SubCmdOptions& _options) const
   }
 
   // -- Retrieve and parse the subcommand options -----------------------------
-  std::vector<std::string> device;
+  std::string device_str;
   std::string plp = "";
```

Comment:
The string class by default initialized the string to an empty string as such there isn't a need for this extra assignment.